Makefile

```diff
@@ -109,7 +109,7 @@ test-advanced:
 
 test-behat:
 	bin/tools sh -c "APP_DEBUG=1 APP_ENV=test bin/post-install-dev.sh"
-	bin/tools vendor/bin/behat --format=progress
+	bin/tools sh -c "APP_DEBUG=0 APP_ENV=test vendor/bin/behat --format=progress"
```

Comment:
can't we put APP_ENV=test in the behat config file ?

Reply:
Not working because the APP_ENV=dev is forced in docker-compose.yml, and the Kernel loads it instead of .env.test file. Do allow behat to boot on test environment, we must revamp the APP_ENV management in the Docker configuration (Dockerfile + docker-compose files)

Reply:
Ok, I'll find a way to override it in another PR